sdk/formrecognizer/Azure.AI.FormRecognizer/src/FormTrainingClient.cs

```diff
@@ -288,14 +288,99 @@ public virtual async Task<Response<AccountProperties>> GetAccountPropertiesAsync
+            var request = new CopyRequest(target._resourceId,
+                                          target._region,
+                                          new CopyAuthorizationResult(target.ModelId, target._accessToken, target.ExpiresOn/*.ToUnixTimeSeconds()*/));
+            ResponseWithHeaders<ServiceCopyCustomModelHeaders> response = await ServiceClient.CopyCustomModelAsync(guid, request, cancellationToken).ConfigureAwait(false);
+            return new CopyModelOperation(ServiceClient, Diagnostics, response.Headers.OperationLocation, target.ModelId);
+        }
+
+        /// <summary>
+        /// Generate authorization for copying a custom model into the target Form Recognizer resource.
+        /// </summary>
+        /// <param name="resourceId">Azure Resource Id of the target Form Recognizer resource where the model will be copied to.</param>
+        /// <param name="region">Location of the target Form Recognizer resource</param>
+        /// <param name="cancellationToken">A <see cref="CancellationToken"/> controlling the request lifetime.</param>
+        /// <returns>A <see cref="Response{T}"/> representing the result of the operation. It can be cast to <see cref="CopyAuthorization"/> containing
+        /// the authorization information neccesary to copy a custom model into a target Form Recognizer resource.</returns>
+        [ForwardsClientCalls]
+        public virtual Response<CopyAuthorization> GetCopyAuthorization(string resourceId, string region, CancellationToken cancellationToken = default)
+        {
+            Argument.AssertNotNullOrEmpty(resourceId, nameof(resourceId));
+            Argument.AssertNotNullOrEmpty(region, nameof(region));
+
+            Response<CopyAuthorizationResult> response = ServiceClient.GenerateModelCopyAuthorization(cancellationToken);
+            return Response.FromValue(new CopyAuthorization(response.Value, resourceId, region), response.GetRawResponse());
+        }
+
+        /// <summary>
+        /// Generate authorization for copying a custom model into the target Form Recognizer resource.
+        /// </summary>
+        /// <param name="resourceId">Azure Resource Id of the target Form Recognizer resource where the model will be copied to.</param>
+        /// <param name="region">Location of the target Form Recognizer resource</param>
+        /// <param name="cancellationToken">A <see cref="CancellationToken"/> controlling the request lifetime.</param>
+        /// <returns>A <see cref="Response{T}"/> representing the result of the operation. It can be cast to <see cref="CopyAuthorization"/> containing
+        /// the authorization information neccesary to copy a custom model into a target Form Recognizer resource.</returns>
+        [ForwardsClientCalls]
+        public virtual async Task<Response<CopyAuthorization>> GetCopyAuthorizationAsync(string resourceId, string region, CancellationToken cancellationToken = default)
```

Comment:
In python we have `region` as `resource_region`. Should we be consistent here? (not sure which one to choose). @kristapratico

Reply:
I feel like `resource_region` is much more descriptive. Why just `region`? @maririos

Reply:
For the object `CopyAuthorization` we landed on `Region` in .NET as all the information comes from the target.
I do agree that for this function, it will be better to have `resourceRegion`. -> will update

Reply:
Sounds good!